draft.markdown

```diff
@@ -0,0 +1,155 @@
+# Statistical Aggregations
+
+* Target Audience: Developers
+* Article Type: Development Frameworks
+
+## External References
+
+* https://www.elastic.co/guide/en/elasticsearch/reference/6.5/search-aggregations-metrics.html
+* https://www.elastic.co/guide/en/elasticsearch/reference/6.5/search-aggregations-metrics-cardinality-aggregation.html
+* https://lucene.apache.org/solr/guide/7_5/the-stats-component.html
+
+## Search Engine Connector Support
+* Elasticsearch 6: Yes
+* Solr 7: Yes
+
+## New/Related APIs
+
+Listed those which are relevant for developers.
+ 
+API (FQCN) | Provided by Artifact |

```

Comment:
@lipusz, did you right-align the firs column on purpose? I'm trying to decide if I like that format or not :thinking:

Reply:
No, I don't know why it happened. Maybe the spaces?

Reply:
It was this line:

> ---------: | :------------------: |

That creates a right-aligned column (the first column) and a centered column (the second one). I thought maybe you'd done it on purpose.

Reply:
:-)

Reply:
I still don't understand why it's producing things to be right-aligned... :stuck_out_tongue:

Reply:
| Tables   |      Are      |  Cool |
|----------|:-------------:|------:|
| col 1 is |  left-aligned | $1600 |
| col 2 is |    centered   |   $12 |
| col 3 is | right-aligned |    $1 |

Source: https://www.tablesgenerator.com/markdown_tables

Reply:
Thanks!

Reply:
oh, it's the `:` that controls. Smart. :smirk: